### PR TITLE
Update COORDINATION.md with Arm review comments

### PR DESCRIPTION
**Pull request by samerhaj**

### CODE REVIEW

File: COORDINATION.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ Initial bootup part covers: Server is Powered On, DPU receives power and starts
...
 Coordinated shutdowns, reboots, crashes, error handling will be details on a separate page.
 
+***SE> My TLDR:
+
+We do not necessarily need to pick one option. 
+
+- Option 1 may be useful as a long-term architected in-band solution, especially with the DOE path (which does not require a PCIe spec change)
+- Option 3 may be preferred by systems that prefer OOB management (BMC manages all devices directly). And it does not necessarily conflict with Option (1) for the in-band host/UEFI operation 
+- Option 2 may be a stop-gap, especially if it does not require a host (HW, BIOS or BMC) changes. But it is not guaranteed to work in all cases.
+
```
Comment (glimchb) [inline COORDINATION.md:20]: merged this section to https://github.com/opiproject/opi-prov-life/blob/main/COORDINATION.md#summary

File: COORDINATION.md [hunk 2]
```diff
@@ -63,6 +79,11 @@ Servers need to change their BIOS/UEFI implementation to accomodate for this opt
    - Accelerator2
    - other
 
+
+***SE> Another thought comes to mind. We could define a new DOE object (which we can do using our own specification with a GUID, without having to go to PCI-SIG). The DOE object can then be formatted to include multiple fields (and versioned/extensible for the future if needed). These fields can include OS_STATUS, CRASHDUMP_STATUS, etc.. registers described above. DOE allows for bi-directional MMIO communication, so we could also have write fields. For more information, refer to the DOE PCIe ECN here: https://members.pcisig.com/wg/PCI-SIG/document/18483?uploaded=1. For adapters that already support DOE hardware (for other features, such as PCIe IDE/CMA, etc..), adding support for a new feature using the same DOE hardware may just require a firmware change (to be confirmed).
```
Comment (glimchb) [inline COORDINATION.md:83]: merged https://github.com/opiproject/opi-prov-life/blob/main/COORDINATION.md#doe-improvement

File: COORDINATION.md [hunk 3]
```diff
@@ -105,6 +135,8 @@ bodies or specs outside of OPI.
 meeting the timing constraints around enumeration.  Dynamic adding and
 deleting devices on the PCI bus via hot plug requires BIOS configuration.
 
+***SE> One thought is that this option can be improved by using a contract/handshake, similar to the UEFI specification method for detecting Adapter Information, such as the Media presence (or other attributes that can be dynamic).  See UEFI spec: https://uefi.org/specs/UEFI/2.10/11_Protocols_UEFI_Driver_Model.html?highlight=adapter_info_#network-media-state . This is already supported by many network adapters and UEFI/BIOS implementations. This removes the need to rely on delay in the driver/OptionROM implementation, and instead gives the control to the BIOS/UEFI to detect when the xPU is ready. This however does NOT address the issues with xPU initialization taking too long that the PCI OptionROM itself does not get loaded in time when the host needs it.
```
Comment (glimchb) [inline COORDINATION.md:138]: merged to https://github.com/opiproject/opi-prov-life/blob/main/COORDINATION.md#improvement

File: COORDINATION.md [hunk 4]
```diff
@@ -169,11 +201,12 @@ Reference:
 
 - <https://www.dmtf.org/dsp/DSP0218> DMTF DSP0218 PLDM for Redfish Device Enablement
 
+
 ### SPDM
 
-SPDM runs over MTCP/I2C, PCIe DOE, and potentually MTCP/USB.  Not that some physical layers do not support Asyncronus Event Notifications (AEN).  SPDM is initiated by the platform BMC to the xPU.
+SPDM runs over MTCP/I2C, PCIe DOE, and potentially MTCP/USB.  Not that some physical layers do not support Asyncronus Event Notifications (AEN).  SPDM is initiated by the platform BMC to the xPU.
```
Comment (glimchb) [inline COORDINATION.md:207]: merged

**Comment by samerhaj** [comment]
Closing as most of this got merged already

**Pull request closed without merging**